### ISSUE TITLE
Track changes (who/when) and expose in UI

### ISSUE THREAD

**Issue by kmcculloch** [edgi-govdata-archiving/archivers.space#49]
If we want to go all in on this, it's probably a phase 2 matter, since it would involve collecting and displaying "meta meta data"--e.g., not just data in the fields, but a trail of when that data was changed and by who. This would be a big effort and veer into "reinventing the wheel" territory--we'd essentially be engineering something like a wiki platform--so it might be better done by leveraging some other kind of back end system down the road. There's also a fair amount of anonymity built into our process--no required email address, anonymous submissions via the Chrome app, etc.--so this kind of audit trail may be of mixed value, particularly if most of our volunteers vanish after attending a single event.

That said, I'm sure it would help, when confronted with an ambiguous set of harvest or research notes, to know who made them so that you could ask for clarification. And if we are serious about the vision of building a robust metadata platform and encouraging domain experts (scientists, etc.) to use it going forward, then we need to consider how to authenticate people's contributions.

In the short term, there are a few things we could do that wouldn't be such a huge effort:

1. We could keep and display a trail of everyone who checked out an item. This wouldn't necessarily show who did exactly what, but it would be a start
2. We could encourage people to sign their comments using their slack handles or some such, or provide dedicated fields for the same

**Comment by danielballan** [comment]
I agree with all of the above. We are already doing (1) -- grep for lockHistory -- and I have a half-baked branch to expose it in the UI under the "Contributors" heading. It involves a join, which Meteor makes really hard to do.
